nodrop/src/lib.rs

```diff
@@ -38,6 +38,14 @@ impl<T> NoDrop<T> {
         NoDrop(Flag::Alive(value))
     }
 
+    /// Create a new **NoDrop** without a value.
+    ///
+    /// It is not safe to use the returned value without first checking `is_alive()`.
+    #[inline]
+    pub unsafe fn dropped() -> NoDrop<T> {
```

Comment:
I'll find use for take too, I think. I'd prefer to skip the dropped constructor if we don't have a use for it yet. just YAGNI it :)